### DISCUSSION TITLE
yarn build failed. node: v22.3.0

### DISCUSSION THREAD

**Discussion by potter-sun** [coral-xyz/backpack#4319] [General]
`
@coral-xyz/provider-injection:build: built provider-injection, with 0 errors
@coral-xyz/background:build: Failed to mark outputs as cached for @coral-xyz/background#build: Unavailable
@coral-xyz/background:build: cache hit, replaying logs 69c7eaadb72b7ff8
@coral-xyz/recoil:build: (node:94620) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
@coral-xyz/recoil:build: (Use `node --trace-deprecation ...` to show where the warning was created)
@coral-xyz/recoil:build: [STARTED] Parse Configuration
@coral-xyz/recoil:build: [SUCCESS] Parse Configuration
@coral-xyz/recoil:build: [STARTED] Generate outputs
@coral-xyz/recoil:build: [STARTED] Generate to ./src/apollo/
@coral-xyz/recoil:build: [STARTED] Load GraphQL schemas
@coral-xyz/recoil:build: [SUCCESS] Load GraphQL schemas
@coral-xyz/recoil:build: [STARTED] Load GraphQL documents
@coral-xyz/recoil:build: [SUCCESS] Load GraphQL documents
@coral-xyz/recoil:build: [STARTED] Generate
@coral-xyz/recoil:build: [FAILED] GraphQL Document Validation failed with 2 errors;
@coral-xyz/recoil:build: [FAILED]   Error 0: Cannot query field "jupiterSwapValidInputTokens" on type "Query".
@coral-xyz/recoil:build: [FAILED]     at /Users/potter/Desktop/wallet/backpack/packages/recoil/src/context/Swap/hooks.ts:3:7
@coral-xyz/recoil:build: [FAILED] 
@coral-xyz/recoil:build: [FAILED] Error 1: Cannot query field "jupiterSwapOutputTokens" on type "Query". Did you mean "jupiterTradableTokens"?
@coral-xyz/recoil:build: [FAILED]     at /Users/potter/Desktop/wallet/backpack/packages/recoil/src/context/Swap/hooks.ts:3:5
@coral-xyz/recoil:build: [SUCCESS] Generate outputs
@coral-xyz/recoil:build: ERROR: command finished with error: command (/Users/potter/Desktop/wallet/backpack/packages/recoil) /private/var/folders/d9/bmbzh5n14v7byzn_zznk3gr80000gn/T/xfs-a62a9c32/yarn run build exited (1)
@coral-xyz/recoil#build: command (/Users/potter/Desktop/wallet/backpack/packages/recoil) /private/var/folders/d9/bmbzh5n14v7byzn_zznk3gr80000gn/T/xfs-a62a9c32/yarn run build exited (1)

 Tasks:    10 successful, 11 total
Cached:    10 cached, 11 total
  Time:    3.215s 
Failed:    @coral-xyz/recoil#build

 ERROR  run failed: command  exited (1)
`

**Comment by justinchitla** [comment]
did you figure out the problem? I'm running into same.
